Plots/Scatter/NCL_scatter_3.py

```diff
@@ -0,0 +1,90 @@
+"""
+NCL_scatter_3.py
+================
+This script illustrates the following concepts:
+    - Drawing a scatter plot over a map
+    - Drawing markers on a map
+    - Attaching markers to a map
+    - Changing the marker color and size in a map plot
+    - Drawing markers on a map indicating the locations of station data
+   
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/scatter_3_1_lg.png and
+                        https://www.ncl.ucar.edu/Applications/Images/scatter_3_2_lg.png
+                         
+"""
+###############################################################################
+# Import packages:
+
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+
+from geocat.viz import util as gvutil
+import geocat.datafiles as gdf
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+
+ds = xr.open_dataset(gdf.get("netcdf_files/95031800_sao.cdf"), decode_times=False)
+lat = ds.lat.isel()
+lon = ds.lon.isel()
+###############################################################################
+# Plot
+
+def Plots(xext, yext, xtic, ytic, size, color):
+    '''
+    Parameters
+    ----------
+    xext : 'tuple'
+        Extent of projection in format (xstart, xend) with values between -180 
+        and 180.
+    yext : 'tuple'
+        Extent of projection in format (ystart, yend) with values between -90 
+        and 90.
+    xtic : 'int'
+        Location of x tick labels instances in format of number between each tick.
+    ytic : 'int'
+        Location of y tick labels instances in format of number between each tick.
```

Comment:
Can you clarify `xtic` and `ytic`? After reading the function, it's clear that this is passed to `range` as the step argument, but I didn't understand that just from reading the comment.

Reply:
Let me know how you feel about the updated version on this one. I'm not sure if I clarified it more, or made it more confusing this time around